lib/jmap/mdn/send/mdn_send_method.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:jmap_dart_client/http/converter/account_id_converter.dart';
+import 'package:jmap_dart_client/http/converter/identities/identity_id_converter.dart';
+import 'package:jmap_dart_client/http/converter/send/send_method_properties_converter.dart';
+import 'package:jmap_dart_client/http/converter/set/set_method_properties_converter.dart';
+import 'package:jmap_dart_client/jmap/core/capability/capability_identifier.dart';
+import 'package:jmap_dart_client/jmap/core/method/request/send_method.dart';
+import 'package:jmap_dart_client/jmap/core/request/request_invocation.dart';
+import 'package:jmap_dart_client/jmap/identities/identity.dart';
+import 'package:jmap_dart_client/jmap/mail/email/submission/set/set_email_submission_method.dart';
+import 'package:jmap_dart_client/jmap/mdn/mdn.dart';
+
+class MDNSendMethod extends SendMethod<MDN> with OptionalOnSuccessUpdateEmail {
```

Comment:
What are ReferenceId and EmailSubmissionId, can you clarify for me, please ?

Reply:
- `ReferenceId` is an id that refers to another id. Defined with the `Prefix` reference keyword and the `Id` value. Example: `#123`
- `EmailSubmissionId` is the id of the `EmailSubmission` method of `Email`